src/status_im/chat/models/input.cljs

```diff
@@ -53,9 +53,11 @@
                 (let [{:keys [commands responses]} (get-in db [:contacts/contacts identity])]
                   (let [commands'  (mapv (fn [[k v]] [k [v :any]]) (merge global-commands commands))
                         responses' (mapv (fn [{:keys [message-id type]}]
-                                           [type [(get responses type) message-id]])
+                                           (when (get responses type)
+                                             [type [(get responses type) message-id]]))
```

Comment:
Super minor point, but this could be written as:
```clj
(when-let [response (get responses type)]
  [type [response message-id]])
```
It's one map access less :)